### PR TITLE
Onboard from dapp popup

### PR DESCRIPTION
**Pull request by jagodarybacka**
Resolves https://github.com/tallycash/extension/issues/2487

### What

Handle case where user wants to connect to dapp but there are no accounts imported to the wallet.
Redirect to onboarding from the dapp popup.

#### Popup based onboarding

https://user-images.githubusercontent.com/20949277/200846037-f9fb7475-0fba-4c06-b517-31b9e04cf1cf.mov

#### Tab based onboarding

https://user-images.githubusercontent.com/20949277/200846101-32b637dd-e230-4e10-aca9-acf22a0f13a0.mov

### Testing

- [ ] remove all accounts from the wallet
- [ ] open any dapp that we are integrated with
- [ ] try to connect the wallet - you should be able to finish onboarding
- [ ] in tabbed version - you have to cancel and reconnect with dapp
- [ ] in popup version - you will be automatically redirected to the dapp connection screen

Latest build: [extension-builds-2578](https://github.com/tallycash/extension/suites/9212939650/artifacts/430109114) (as of Wed, 09 Nov 2022 13:40:53 GMT).

### CODE REVIEW

**Comment by jagodarybacka** [comment]
@VladUXUI please take a look at how it is working with tabbed and popup versions 😊

File: ui/pages/DAppConnectRequest.tsx [hunk 1]
```diff
@@ -35,36 +36,35 @@ export default function DAppConnectRequest(): ReactElement {
...
-      </div>
-    )
+  if (typeof permission === "undefined") {
+    // something went wrong with permisison request
+    return <ErrorFallback />
+  }
+
+  if (typeof currentAccountTotal === "undefined") {
+    // there is no account, let's onbaord first
+    return <Redirect to="/onboarding/info-intro" />
```
Comment (hyphenized) [inline ui/pages/DAppConnectRequest.tsx:67]: Non blocking but I think it'd be more appropriate to handle the redirection before we reach this route by removing the exception [added here](https://github.com/tallycash/extension/blob/4738212c69bb7e296e6beb63567ece3f0d4f6165/ui/pages/Popup.tsx#L226)

**Review by hyphenized — APPROVED** [review]
Works well 👍

File: background/services/provider-bridge/show-popup.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ export default async function showExtensionPopup(
 ): Promise<browser.Windows.Window> {
   const { left = 0, top, width = 1920 } = await browser.windows.getCurrent()
   const popupWidth = 384
-  const popupHeight = 600
+  const popupHeight = 628
```
Comment (greg-nagy) [inline background/services/provider-bridge/show-popup.ts:9]: ❤️

**Review by greg-nagy — APPROVED** [review]
looks good, works well 

let's 🚚

**Pull request merged**